packages/boot/tools/liquidation.ts

```diff
@@ -130,21 +91,22 @@ export const makeLiquidationTestContext = async t => {
     const managerPath = `published.vaultFactory.managers.manager${managerIndex}`;
     const { advanceTimeBy, readLatest } = swingsetTestKit;
 
-    !priceFeedDrivers[collateralBrandKey] ||
-      Fail`setup for ${collateralBrandKey} already ran`;
-    priceFeedDrivers[collateralBrandKey] = await makePriceFeedDriver(
-      collateralBrandKey,
-      agoricNamesRemotes,
-      walletFactoryDriver,
-      // TODO read from the config file
-      [
-        'agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr',
-        'agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8',
-        'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
-        'agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p',
-        'agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj',
-      ],
-    );
+    await 0;
+    if (!priceFeedDrivers[collateralBrandKey]) {
+      priceFeedDrivers[collateralBrandKey] = await makePriceFeedDriver(
+        collateralBrandKey,
+        agoricNamesRemotes,
+        walletFactoryDriver,
+        // TODO read from the config file
```

Comment:
maybe a good time to do this

Reply:
That'd be great, but once again not part of what this PR is trying to do.